SNAP/README.md

```diff
@@ -0,0 +1,67 @@
+# Supplemental Nutrition Assistance Program Imputation
+
+SNAP, previously known as Food Stamps,is reported on a household, rather than
+individual, level in the CPS. Therefore, each member of the household will report
+the same benefit amounts and time receiving benefits. For example, if one
+person in the household reports receiving $1,000 in benefits for four months,
+everyone in the household will report the same.
+
+Target data for the imputation came from official SNAP data. In Fiscal Year 2014,
+an average of 22.74 million households claimed about $70 billion in benefits.
+
+## Imputation Procedure
+
+Recipients and benefits are imputed using a two step procedure. First, participation
+is imputed to match administrative totals. Second, we impute an adjustment ratio
+for benefits in order to match total benefit amounts.
+
+## Imputing Recipients
+
+A basic linear regression is used to predict the likelihood of participation in
+the SNAP program. In compliance with SNAP rules, only net income is used as an
+independent variable. Net income is computed using the following rules, also in
+compliance with program rules:
+1. Gross income for each household is attained by summing up the earned and
+unearned income of all household members.
+2. A deduction of 20% of earned income is applied.
+3. An additional deduction is applied according to household size
+
+|Size | Deduction|
+|------|----------|
+| <= 3| $155 |
+| 4 | $168 |
+| 5  | $197 |
+| >= 6| $ 226|
+
+4. For households with members ages 60 or older, medical expenses exceeding $35,
+if they are not paid by insurance or someone else, are also deducted
+5. Child support can be deducted
+6. Dependent care and shelter costs may also be deducted. We use the average
+official numbers of $10 and $290 a month. These costs are not provided by the
```

Comment:
Looking at how many hard-coded numbers in the scripts, I'm thinking may be we should add a section what need to be updated when moving on to next year? What do you think?